release/202008/metrics-head-rc.md

```diff
@@ -0,0 +1,51 @@
+
+To obtain a pdf copy of previous releases or see what is new in this release please visit the [release notes page](https://chaoss.community/release-history/).
+
+To make suggestions or edits to this webpage please visit the [website repo](https://github.com/chaoss/website) and open an issue or create a pull request.
+
+#### Disclaimer
+
+Released metrics are only a subset of metric ideas that are being developed. CHAOSS makes no assertions about the completeness of this list. If you would like to learn more about metrics, suggest new metrics, and or help define metrics please visit the working group repositories.  
+
+The CHAOSS project recognizes that there are ethical and legal challenges when using the metrics and software provided by the CHAOSS community. The particular challenges arise in the use that is specific to your context.  
+
+* Ethical challenges exist around protecting community members and empowering them with their personal information.  
+
+* Legal challenges exist around GDPR and similar laws or regulations that protect personal information of community members.  
+
+## Focus Areas by Working Group  
+
+ *CHAOSS metrics are sorted into Focus Areas. CHAOSS uses a Goal-Question-Metric format to present metrics. Individual metrics are released based on identified goals and questions. The metrics include a detail page with definitions, objectives, and examples.*  
+
+#### Common Metrics WG  
+* [What](#user-content-focus-area---what)  
+* [When](#user-content-focus-area---when)  
+* [Who](#user-content-focus-area---who)  
+#### Diversity and Inclusion WG  
+* [Event Diversity](#user-content-focus-area---event-diversity)  
+* [Governance](#user-content-focus-area---governance)  
+* [Leadership](#user-content-focus-area---leadership)  
+#### Evolution WG  
+* [Code Development Activity](#user-content-focus-area---code-development-activity)  
+* [Code Development Efficiency](#user-content-focus-area---code-development-efficiency)  
+* [Code Development Process Quality](#user-content-focus-area---code-development-process-quality)  
+* [Issue Resolution](#user-content-focus-area---issue-resolution)  
+* [Community Growth](#user-content-focus-area---community-growth)  
+#### Risk WG  
+* [Business Risk](#user-content-focus-area---business-risk)  
+* [Code Quality](#user-content-focus-area---code-quality)  
+* [Licensing](#user-content-focus-area---licensing)  
+* [Security](#user-content-focus-area---security)  
+#### Value WG  
+* [Ecosystem Value](#user-content-focus-area---ecosystem-value)  
+* [Labor Investment](#user-content-focus-area---labor-investment)  
+* [Living Wage](#user-content-focus-area---living-wage)  
+
+### Important Dates for Release 202008  
+202008 Release Freeze: July 1st, 2020    
+202008 Public Comment Period: July 1st, 2020 to July 31st, 2020  
+202008 Official Metrics Release Date: August 1st, 2020  
```

Comment:
We need more time to prepare the release than 1 day.
I propose we announce the review period tomorrow (June 29) and leave ourselves three days after the review period to prepare the release.

```suggestion
202008 Release Freeze: June 24, 2020    
202008 Public Comment Period: June 29, 2020 to July 28, 2020  
202008 Official Metrics Release Date: August 1, 2020  
```

Reply:
We could also just announce the release as the first week of august. That would give us a little flexibility.